js/views/callinfoview.js

```diff
@@ -91,6 +111,7 @@
 				canFullModerate: this._canFullModerate(),
 				isPublic: this.model.get('type') === 3,
 				showShareLink: !canModerate && this.model.get('type') === 3,
+				canPublish: OCA.SpreedMe.canPublish(),
```

Comment:
`TypeError: OCA.SpreedMe.canPublish is not a function`

Reply:
This is from another pull request https://github.com/nextcloud/spreed/pull/1298